purchase_security/security/security.xml

```diff
@@ -47,5 +55,13 @@
             >['|',('order_id.user_id','=',user.id),('order_id.user_id','=',False)]</field>
             <field name="groups" eval="[(4,ref('group_purchase_own_orders'))]" />
         </record>
+        <record model="ir.rule" id="purchase_order_user_group_purchases_rule">
+            <field name="name">View purchase orders (Group purchases)</field>
```

Comment:
Follow the nomenclature of the rest of the groups:

![Selección_037](https://github.com/OCA/purchase-workflow/assets/7165771/acde18a2-fa19-438a-a7e7-add4198ebfd8)


```suggestion
            <field name="name">User (team orders)</field>
```

And even more, you have broken the inheritance, as you have to change the "Administrator" to inherit from this new group.

Reply:
Thanks for all the suggestions @pedrobaeza , now I have a better understanding of the approach to be taken. I made some modifications, and I am aware that i have not added yet some view for the new model. Review my changes when you have time please.